src/Automation/Automation.csproj

```diff
@@ -99,7 +102,7 @@
     <None Include="ReadMe.md" />
   </ItemGroup>
   <ItemGroup>
-    <DropSignedFile Include="$(OutDir)\Axe.Windows.Automation.dll" />
+    <DropSignedFile Include="bin\Debug\\Axe.Windows.Automation.dll" />
```

Comment:
Was this change to use the Debug file intentional?

Reply:
Seems odd to me

---
In reply to: [293140526](https://github.com/microsoft/axe-windows/pull/53#discussion_r293140526) [](ancestors = 293140526)